portfolio/src/main/webapp/style.css

```diff
@@ -231,4 +231,99 @@ body {
   .timeline > li.timeline-inverted > .timeline-panel {
     padding: 0 20px 20px;
   }
+}
+
+.slides {
+  display: none;
```

Comment:
Please use alphabetical order for CSS declarations within a selector as per style guid.

Also `max-width: 600px`